### PR TITLE
Add doc for clearML integration

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from super_gradients.training import Trainer, models
```
Comment (ofrimasad) [inline src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py:1]: why is this under deci_logger_example? it is a clearml logger

**Review by ofrimasad — COMMENTED** [review]
just fix the example file location please

File: src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py:1]: For some reason the folder was not renamed, but I thought about having a single folder "loggers_examples" that would include both deci_logger_example and clearml_logger_example.
What do you think?

Comment (ofrimasad) [reply src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py:1]: sounds ok. rename the folder please

Comment (Louis-Dupont) [reply src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py]: renamed to `loggers_examples` in 397596cb6620de96b3a88f5f56c33f0d75a52af1

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**